Src/FluentAssertions/Collections/GenericCollectionAssertions.cs

```diff
@@ -813,10 +813,15 @@ public AndConstraint<TAssertions> Contain(IEnumerable<T> expected, string becaus
     /// Asserts that a collection of objects contains at least one object equivalent to another object.
     /// </summary>
     /// <remarks>
-    /// Objects within the collection are equivalent to the expected object when both object graphs have equally named properties with the same
-    /// value, irrespective of the type of those objects. Two properties are also equal if one type can be converted to another
-    /// and the result is equal.
+    /// <para>
+    /// By default, objects within the collection are seen as equivalent to the expected object when both object graphs have equally named properties with the same
+    /// value, irrespective of the type of those objects.
     /// Notice that actual behavior is determined by the global defaults managed by <see cref="AssertionOptions"/>.
+    /// </para>
+    /// <para>
+    /// <strong>Important:</strong> You cannot use this method to assert whether a collection contains a subset of another collection.
+    /// Use e.g. <see cref="BeSubsetOf"/> instead.
```

Comment:
I read "collection contains a subset of another collection" if `subject` and `expectation` has a non-empty intersection.

To ask if `subject` contains at least the items in `expectation` is to ask whether `subject` is a superset of `expectation`.
In FA that is `subject.Should().Contain(expectation)`.
But `Contain` compares items by equality and not equivalence.

Reply:
What do you mean? Should I change `BeSubsetOf` to `Contain`?

I previously wrote something like: Use `BeSubsetOf` instead using the complete collections as expectation.

Reply:
What people seems to be thinking `subject.Should().ContainEquivalentOf(expectation)` does is checking if `expectation` is a subset of `subject`, where elements are compared by equivalency.
![image](https://github.com/fluentassertions/fluentassertions/assets/919634/ffcc03a9-6a45-4e58-afe0-95cccb6a21d7)
(I conclude this also by a first-hand experience with a colleague)

This is how I read "a collection contains a subset of another collection"
![image](https://github.com/fluentassertions/fluentassertions/assets/919634/a902ad3a-be8e-453c-9534-c0fe864bc25d)


`subject.Should().BeSubsetOf(expectation)` is the opposite of what people seems to seek.
![image](https://github.com/fluentassertions/fluentassertions/assets/919634/7906ac05-1235-4fc0-97e5-6ab9d761cde0)


Currently FA does not contain a method to assert that an expectation of elements is equivalent to a subset of the subject.
`subject.Should().Contain(expectation)` is the closest we offer, but this compares elements by equality and not equivalency.


I think either of these would be a bit more precise.
(And I'm fine with either)
```cs
/// <b>Important:</b> You cannot use this method to assert whether a subset of the collection is equivalent to the <paramref name="expectation"/>.
/// <b>Important:</b> You cannot use this method to assert whether the <paramref name="expectation"/> is equivalent to a subset of the collection.
```

Btw, I really like your note about a "single item" 👍 
It's much more applicable to most programmers than some formal definition about a [surjective function](https://en.wikipedia.org/wiki/Surjective_function) between the subject and the expectation.